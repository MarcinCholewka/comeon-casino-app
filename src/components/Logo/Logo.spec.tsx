import { render } from '@testing-library/react';
import { Logo } from './Logo';

describe('<Logo />', () => {
  it('should render without crashing', () => {
    const { container, getByAltText } = render(<Logo />);

    expect(container).toMatchSnapshot();
    expect(getByAltText('logo')).toBeInTheDocument();
  });
});
