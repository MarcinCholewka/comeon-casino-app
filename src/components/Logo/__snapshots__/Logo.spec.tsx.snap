// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render without crashing 1`] = `
<div>
  <div
    class="ui one column center aligned page grid"
  >
    <div
      class="column twelve wide"
    >
      <img
        alt="logo"
        src="src/assets/logo.svg"
      />
    </div>
  </div>
</div>
`;
